tcf_website/templates/schedule/add_course_modal.html

```diff
@@ -0,0 +1,108 @@
+{% load static %}
+
+{% block styles %}
+
+{% endblock %}
+
+<div class="modal fade" id="addCourseModal" tabindex="-1" role="dialog" aria-labelledby="addCourseModal" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+
+        <div class="modal-header">
+          <h5 class="modal-title">Select a Section for <strong>{{course}}</strong></h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div id="second-modal-body" class="modal-body">
+
+
+        </div>
+        <div class="modal-footer">
+            <button id="section_select_btn" form="select_section_form" type="submit" value="Submit" class="btn btn-primary" disabled >
+                Confirm
+            </button>
+        </div>
+      </div>
+    </div>
+</div>
+
+<script src="{% static 'schedule/parse_time.js' %}"></script>
+<script>
+
+  function buttonDoubleSubmit() {
+    // this function is for preventing a double submit
```

Comment:
What do you mean by this - "double-submit" on forms is usually handled via a sort of threshold/debounce time.

Reply:
This code basically does that.